src/app/data/highway.py

```diff
@@ -137,21 +139,41 @@ def queries_no_cycleway(args):
 
     def queries_no_cycleway_no_parking(args):
         """
-        Removes cycleway tags from queries and retrieves the query result
+        Removes cycleway and parking tags from queries and retrieves the query result
         """
         infra_types = args[0]
         country = args[1]
         city = args[2]
         res = api.get(
-            "area[name = " + country + "]->.country; area[name = " + city + "]->.city; "
+            f"area[name = '{country}']->.country; area[name = '{city}']->.city; "
             "(way" + infra_types + "(area.city)(area.country); "
             "- "
             "(way[~'^cycleway:.*$'~'.'](area.city)(area.country); "
             "way[~'^cycleway.*$'~'.'](area.city)(area.country);"
-            "way[~'^parking:.*$'~'.'](area.city)(area.country);););",
+            "way[~'^parking:.*$'~'.'](area.city)(area.country);"
+            "way[~'^parking.*$'~'.'](area.city)(area.country);););",
             responseformat="json")
         return {f'{infra_types}[!cycleway][!parking]': res["elements"]}
 
+    def queries_no_parking(args):
+        """
+        Removes parking tags from queries and retrieves the query result
+        """
+        infra_types = args[0]
+        country = args[1]
+        city = args[2]
+        elements = []
+        for infra_type in infra_types:
+            elements += api.get(
+                    "area[name = " + country + "]->.country; area[name = " + city + "]->.city; "
```

Comment:
Maybe rewrite this to use f-strings as well (uniformity)

Reply:
hmmm very weird. I forgot this and normally this would give me an error, idk why it didn't